### DISCUSSION TITLE
Can we make the project bigger for a better usage ?

### DISCUSSION THREAD

**Discussion by hwpplayer1** [palkan/logidze#190] [Q&A]
Hi there, I need to contribute to your project, thus I can make my habits or skills better and smarter

Shouldn't we need a GitHub static page for documentation ? I can start over there ...

Thanks a lot See you then

**Comment by palkan** [comment]
Hey!

> Shouldn't we need a GitHub static page for documentation ?

Not sure. How is this different form having a Readme?

**Reply by hwpplayer1** [reply]
It is a collection of all README features
